src/interfaces/name-check-interfaces/name-check-model-interface.ts

```diff
@@ -6,6 +6,7 @@ export interface NameCheckModelIF {
     analyzeStructurePending: boolean
     conflictsExact: Array<string>
     conflictsConditional: Array<string>
+    conflictsConditionalInstructions: Array<{ word: string, instructions: string }>
```

Comment:
I know tedious and simple, might be best to use an interface for all instances of this object.

Reply:
haha yes I will add that in